lib/src/game/field.dart

```diff
@@ -69,16 +69,17 @@ class Field extends Array2d<bool> {
   int getAdjacentCount(int x, int y) {
     assert(!get(x, y), 'Cannot get adjacent count from populated list!');
 
-    var val = _adjacents.get(x, y);
+    final val = _adjacents.get(x, y);
 
     if (val == null) {
-      val = 0;
+      var newVal = 0;
```

Comment:
I think this is because of https://github.com/dart-lang/sdk/issues/40011 .  Otherwise I think the original code should have worked.  cc @scheglov @stereotype441

Reply:
I found this annoying! Glad it's tracked!